com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -796,7 +796,14 @@ internal void InvokeBehaviourNetworkSpawn()
             for (int i = 0; i < ChildNetworkBehaviours.Count; i++)
             {
                 ChildNetworkBehaviours[i].InternalOnNetworkSpawn();
-                ChildNetworkBehaviours[i].OnNetworkSpawn();
+                try
+                {
+                    ChildNetworkBehaviours[i].OnNetworkSpawn();
+                }
+                catch (Exception e)
+                {
+                    Console.WriteLine(e);
```

Comment:
Do we really want to use Console? What does Unity use elsewhere? Is it not Debug.LogException?

Reply:
+1 on Albin's comment, and same applies below too.

Reply:
Uh... huh. Why did I type that? Console.WriteLine is obviously wrong. Thank you for pointing this out.